platform/api/aws/images.go

```diff
@@ -333,6 +334,42 @@ func (a *API) CreateHVMImage(snapshotID string, diskSizeGiB uint, name string, d
 	return a.createImage(params)
 }
 
+func (a *API) AddLaunchPermission(imageid string) ([]byte, error) {
+	mod := &ec2.ModifyImageAttributeInput{
+		ImageId: &imageid,
+		LaunchPermission: &ec2.LaunchPermissionModifications{
+			Add: []*ec2.LaunchPermission{
+				{
+					Group: aws.String("all"),
+				},
+			},
+		},
+	}
+	resp, err := a.ec2.ModifyImageAttribute(mod)
+	if err != nil {
+		return []byte{}, err
+	}
+	return json.Marshal(resp)
```

Comment:
Personal opinion: I tend to wrap the errors whenever it's possible to bring more context and to identify if the issue appears in the `ModifyImageAttribute` or in the `Marshal` of the value.
```suggestion
	resp, err := a.ec2.ModifyImageAttribute(mod)
	if err != nil {
		return []byte{}, fmt.Error("modifying image attribute: %w, err)
	}
	
	res, err := json.Marshal(resp)
	if err != nil {
		return []byte{}, fmt.Error("marshaling the API response: %w, err)
	}
	
	return res, nil
```

Reply:
ok, added to RemoveLaunchPermission()